### PR TITLE
fix: support manta parachain fee calculations

### PR DESCRIPTION
**Pull request by grenade**
may be tested against any of: 

- wss://kuhlii.manta.systems
- wss://melanyae.manta.systems
- wss://munkiana.manta.systems
- wss://pectinata.manta.systems
- wss://tarapacana.manta.systems
- wss://thurstoni.manta.systems

### CODE REVIEW

File: calc/build.sh [hunk 1]
```diff
@@ -8,3 +8,7 @@ else
     echo "Fee debugging enabled"
     wasm-pack build --debug --target nodejs --scope substrate "$SCRIPT_DIR" -- --features debug
 fi
+# append a newline to pkg/package.json if it isn't there
+if [ "$(tail -c1 "$SCRIPT_DIR/pkg/package.json"; echo x)" != $'\nx' ]; then
```
Comment (TarikGul) [inline calc/build.sh:12]: 👍

**Comment by TarikGul** [comment]
Thanks for the PR and contribution, looks good so far, just going to run it locally as well.

**Review by TarikGul — APPROVED** [review]
lgtm 👍  Ran it locally, and it all builds correctly!

**Review by insipx — APPROVED** [review]

**Comment by TarikGul** [comment]
Just give the thumbs up when you are happy with this to merge. @grenade

**Comment by grenade** [comment]
> Just give the thumbs up when you are happy with this to merge. @grenade

we're happy for this to be merged!

**Pull request merged**